source/jormungandr/tests/routing_tests_new_default.py

```diff
@@ -99,6 +99,57 @@ def test_first_walking_last_walking_section_mode(self):
         assert response["journeys"][0]["type"] == "best"
         assert "walking" in response["journeys"][0]["tags"]
 
+    def test_first_ridesharing_last_walking_section_mode(self):
+        query = "journeys?from=0.0000898312;0.0000898312&to=0.00188646;0.00071865&datetime=20120614T075500&"\
+                "first_section_mode[]={first}&last_section_mode[]={last}"\
+                .format(first='ridesharing', last='walking')
+        response = self.query_region(query)
+        check_best(response)
+        self.is_valid_journey_response(response, query)
+        assert len(response["journeys"]) == 3
+        assert response["journeys"][0]["type"] == "best"
+        assert "walking" in response["journeys"][0]["tags"]
+        rs_journey = response["journeys"][2]
+        assert "ridesharing" in rs_journey["tags"]
```

Comment:
Some of these tests ought to be mutualized when managing ridesharing fallback... Now is too soon I guess